optimade/validator/utils.py

```diff
@@ -214,29 +223,39 @@ def get(self, request: str):
 
         status_code = None
         retries = 0
+        errors = []
         # probably a smarter way to do this with requests, but their documentation 404's...
         while retries < self.max_retries:
             retries += 1
             try:
-                self.response = requests.get(self.last_request, headers=self.headers)
-            except requests.exceptions.ConnectionError as exc:
-                sys.exit(
-                    f"{exc.__class__.__name__}: No response from server at {self.last_request}, please check the URL."
+                self.response = requests.get(
+                    self.last_request,
+                    headers=self.headers,
+                    timeout=(self.timeout, DEFAULT_READ_TIMEOUT),
```

Comment:
How is a tuple a valid value for `timeout` here? Shouldn't it just be:
```suggestion
                    timeout=self.timeout,
```

Reply:
https://docs.python-requests.org/en/master/user/quickstart/#timeouts

There are two timeouts, connect and read. If you provide one, it overrides both, which means we end up with too small a read timeout for the slow implementations

Reply:
that's probably what I've then unknowingly set for the requests with timeouts I've done to materials cloud...